contrib/packaging/rpm/cfg/Dockerfile

```diff
@@ -1,24 +1,13 @@
-FROM fedora:24
+FROM fedora:25
 
-MAINTAINER "Andre Martins <andre@cilium.io>"
+LABEL "Maintainer: Marcin Skarbek <marcin@skarbek.name>"
 
-RUN dnf install -y gettext curl gcc fedora-packager fedora-review && \
-cd /tmp && \
-curl -Sslk -o go1.7.4.linux-amd64.tar.gz \
```

Comment:
I'm fine using official golang packages if they work. The only reason we can't use clang 1.8.x right now is that it produces bytecode which older kernels don't understand. Sticking to clang 1.7.x forever is also not a good option. We will shift to clang 1.8.x when fedora starts shipping kernel 4.9.17 which is required for clang 1.8.x.